ajax/libs/slickgrid/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "slickgrid",
+  "version": "2.3.4",
+  "description": "A lightning fast JavaScript grid/spreadsheet",
+  "filename": "slick.core.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/6pac/SlickGrid.git"
+  },
+  "keywords": [
+    "slickgrid",
+    "grid"
+  ],
+  "author": {
+    "name": "Michael Leibman",
+    "email": "michael.leibman@gmail.com"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/mleibman/SlickGrid/wiki",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/6pac/SlickGrid.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "lib/*",
```

Comment:
if cdnjs doesn't hold those libs, we should host them, too. If they has git repo or npm package, send another PR to add them, and mention this issue in the PR. If they don't adding them with this lib is fine.